tools/lsp/ui/draw-area.slint

```diff
@@ -74,6 +75,16 @@ component SelectionFrame {
             border-width: 1px;
             background: parent.resizing || parent.moving ? root.selection.border-color.with-alpha(0.5) : root.selection.border-color.with-alpha(0.0);
         }
+
+        key-handler := FocusScope {
```

Comment:
Are you sure that you want an extra click to focus here? Or perhaps it would make more sense to call focus() when the selection frame is clicked on?

Reply:
We indeed have to make sure this plays well with the focus within the preview. (that one can't have focus in the preview)